SWEgit/src/com/company/Main.java

```diff
@@ -198,4 +200,15 @@ public static void GetAverage(String[] arr) {
         System.out.println("Average : "+sum/arr.length);
         }
     }
+	
+    public static <T> void ReverseArray(T[] array) {
```

Comment:
Please take copy from the array and change in it .... when we excute all the original array will be changed in you function so please take copy from the array and change it